appf.adoc

```diff
@@ -41,7 +41,7 @@ __Map coordinates:__:: The x (abscissa) and y (ordinate) rectangular coordinates
 __Notes:__:: Notes on using the **`PROJ`** software package for computing the mapping may be found at
 link:$$https://proj.org/operations/projections/aea.html$$[https://proj.org/operations/projections/aea.html]
 and
-link:$$http://geotiff.maptools.org/proj_list/albers_equal_area_conic.html$$[http://geotiff.maptools.org/proj_list/albers_equal_area_conic.html].
+link:$$https://geotiff.maptools.org/proj_list/albers_equal_area_conic.html$$[https://geotiff.maptools.org/proj_list/albers_equal_area_conic.html].
```

Comment:
Throughout this appendix (Appendix F) file we have various links with the root URL 'https://geotiff.maptools.org'. All such links will hang when the HTTPS protocol is used. Therefore it looks like we can't specify HTTPS for these. I suggest we document this as a known exception to the rule of 'always use HTTPS for links across CF Conventions documents and infrastructure' as alluded to in #463.

Reply:
I totally agree with your suggestion. And the root url geotiff.maptools.org (irrespective of http:// or https://) redirects to https://github.com/OSGeo/libgeotiff, which suggests that the information is not updated. This suggests that we might want to consider to find an alternative source to link to. But that is for another issue to deal with.

Reply:
Thanks Lars. I will open another Issue concerning the out-of-date nature of the GeoTiff links, in that case. And for now and for this PR, I will revert the GeoTiff links to specifying HTTP and not HTTPS. I'll do both this afternoon.

As for the suggestion which you second:

> I suggest we document this as a known exception to the rule of 'always use HTTPS for links across CF Conventions documents and infrastructure' as alluded to in https://github.com/cf-convention/cf-conventions/issues/463.

shall we open a new issue for this? Or can this be considered a natural part of this Issue/PR pair? I imagine to become a strict rule we might need governance permission, too, hence should ask others to confirm they are happy with it.

Reply:
Well, I don't think we need a strict rule for this. To my [limited] knowledge https is what is used nowadays, but occasionally old sites have not been updated. But a separate issue regarding the geotiff links would be useful.

Reply:
Fair enough, maybe it just comes under generally understood good practice/principles.

Reply:
I reverted these GeoTiff links back to HTTP in [205a321](https://github.com/cf-convention/cf-conventions/pull/476/commits/205a3213f5f45c85afcad8bef1e1b6a08debb5f6).

Opening the Issue now. Then I will mark this as resolved.

Reply:
Issue now opened for this: see #479, so I am resolving this comment.